src/pages/search/index.tsx

```diff
@@ -86,26 +80,18 @@ const Form = (props: IFormProps): ReactElement => {
         <div className="col-span-6">
           <div className="flex items-center space-x-3">
             <div className="flex-1">
-              <SearchBarWrapper searchService={searchService} />
+              <SearchBar isLoading={isFetching} />
             </div>
           </div>
-          <NumFound searchService={searchService} count={result.numFound} />
-          {/* <Filters /> */}
+          {!isFetching && <NumFound count={data?.numFound ?? 0} />}
         </div>
         <div className="col-span-6">
-          {isFailure || typeof serverError === 'string' ? (
+          {isSuccess && <SimpleResultList docs={data.docs} indexStart={params.start} />}
```

Comment:
I'm debating how to handle these.  I think we may be better off keeping that action panel out of the result list component.  That way we can keep it simple, and it can be used in other places without having to turn actions on/off since they probably only make sense in one spot.

I'm open to suggestions on this, of course!